src/main/kotlin/org/bitcoindevkit/karavan/Karavan.kt

```diff
@@ -35,4 +33,12 @@ class WalletController() {
         val newAddress = wallet.getNewAddress()
         return "hello $name, your new address is $newAddress"
     }
+
+    @PutMapping("/open_wallet")
```

Comment:
Is there a reason to map to `@PutMapping("/open_wallet)` rather than `@PutMapping` ? In #9 I was expecting the full REST API to be `PUT /wallet`.